plugins/chat/config/routes.rb

```diff
@@ -17,6 +17,8 @@
     post "/channels/:channel_id/memberships/me" => "channels_current_user_membership#create"
     put "/channels/:channel_id/notifications-settings/me" =>
           "channels_current_user_notifications_settings#update"
+    put "/channels/read/" => "reads#update_all"
```

Comment:
Sorry Im nitpicking here, but the `_all` is not needed IMO

Reply:
But they both go to the same controller, are you saying they should still both be in the same endpoint with the switching based on params like I had before?

Reply:
I was thinking two controllers... but you are right that might be weird, ok :+1:

Reply:
Sweet, will leave as-is